decompiler/util.py

```diff
@@ -433,18 +433,18 @@ def __init__(self, needs_space, reorderable=False):
         self.reorderable = reorderable
 
     def append(self, *args):
-        self.words.extend(filter(None, args))
+        self.words.extend([_f for _f in args if _f])
```

Comment:
Why this change ? filter is good, and I think it works on this instance

Reply:
I'd argue with your statement. Filer is more clear method and for those who is unfamiliar with it, I'd recommend to read `help(filter)` at least, for further reading Python tutorial and library reference. 

Additionally, filter is optimised to be fast

Reply:
If it's faster then its a good argument and we have two votes against the generator. I'm not a hardass and sit on it just because. Even if i like the gen way. Going to revert. Thanks folks